sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/FacetResult.java

```diff
@@ -1,61 +1,73 @@
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
 
 package com.azure.search.documents.models;
 
 import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.HashMap;
 import java.util.Map;
 
-/**
- * A single bucket of a facet query result. Reports the number of documents
- * with a field value falling within a particular range or having a particular
- * value or interval.
- */
+/** The FacetResult model. */
 @Fluent
 public final class FacetResult {
-    /*
-     * Unmatched properties from the message are deserialized this collection
-     */
-    @JsonProperty(value = "")
-    private Map<String, Object> additionalProperties;
-
     /*
      * The approximate count of documents falling within the bucket described
      * by this facet.
      */
     @JsonProperty(value = "count", access = JsonProperty.Access.WRITE_ONLY)
     private Long count;
 
+    /*
+     * A single bucket of a facet query result. Reports the number of documents
+     * with a field value falling within a particular range or having a
+     * particular value or interval.
+     */
+    @JsonIgnore private Map<String, Object> additionalProperties;
+
+    /**
+     * Get the count property: The approximate count of documents falling within the bucket described by this facet.
+     *
+     * @return the count value.
+     */
+    public Long getCount() {
+        return this.count;
+    }
+
     /**
-     * Get the additionalProperties property: Unmatched properties from the
-     * message are deserialized this collection.
+     * Get the additionalProperties property: A single bucket of a facet query result. Reports the number of documents
+     * with a field value falling within a particular range or having a particular value or interval.
      *
      * @return the additionalProperties value.
      */
+    @JsonAnyGetter
     public Map<String, Object> getAdditionalProperties() {
         return this.additionalProperties;
     }
 
     /**
-     * Set the additionalProperties property: Unmatched properties from the
-     * message are deserialized this collection.
+     * Set the additionalProperties property: A single bucket of a facet query result. Reports the number of documents
+     * with a field value falling within a particular range or having a particular value or interval.
      *
      * @param additionalProperties the additionalProperties value to set.
      * @return the FacetResult object itself.
      */
-    FacetResult setAdditionalProperties(Map<String, Object> additionalProperties) {
+    public FacetResult setAdditionalProperties(Map<String, Object> additionalProperties) {
```

Comment:
Is this class a facet query result? if so, we should only explore the getter to user but not the setter.

Reply:
Good catch, I'll create an APIView for this change given the amount of code that is being moved around due to Swagger generation changes.

Reply:
Going to leave this as is as the Map is mutable using the getter so there is no big reason to hide the setter.